src/ShareRoutes.test.jsx

```diff
@@ -1,13 +1,40 @@
-import React from 'react'
-import {render} from '@testing-library/react'
-import {mockRoutes} from './BaseRoutesMock.test'
-import ShareRoutes from './ShareRoutes'
+import {isValidModelURL, constructModelPath} from './ShareRoutes'
 
 
-test('ShareRoutes', () => {
-  const {getByText} = render(mockRoutes(
-      <ShareRoutes
-        installPrefix={'/'}
-        appPrefix={'share'} /> ))
-  expect(getByText(/BLDRS/i)).toBeInTheDocument()
+// valid url format
```

Comment:
The first arg for test is the description, so just move the comment into the first argument.  Sometimes I'll do the name of the method if it's just one test.  If it's multiple, like here, you can move your comment in after the method name e.g. "isValidModelURL: valid format"